tests/profiles/stat_test.py

```diff
@@ -151,11 +151,7 @@ def test_check_italic_axis_in_stat():
     # Remove ital axes
     for i, font in enumerate(fonts):
```

Comment:
Now you don't need the `i` variable anymore. So also no need for the `enumerate`.

Reply:
Done